--- cbor2-5.2.0.post1/tests/test_decoder.py.orig	2021-12-22 23:33:04.000000000 +0100
+++ cbor2-5.2.0.post1/tests/test_decoder.py	2024-05-19 12:10:34.423331549 +0200
@@ -430,8 +430,8 @@ def test_uuid(impl):
 
 @pytest.mark.skipif(sys.version_info < (3, 3), reason="Address decoding requires Py3.3+")
 @pytest.mark.parametrize('payload, expected', [
-    ('d9010444c00a0a01', ip_address('192.10.10.1')),
-    ('d901045020010db885a3000000008a2e03707334', ip_address('2001:db8:85a3::8a2e:370:7334')),
+    ('d9010444c00a0a01', ip_address(u'192.10.10.1')),
+    ('d901045020010db885a3000000008a2e03707334', ip_address(u'2001:db8:85a3::8a2e:370:7334')),
     ('d9010446010203040506', (260, b'\x01\x02\x03\x04\x05\x06')),
 ], ids=[
     'ipv4',
@@ -459,9 +459,9 @@ def test_bad_ipaddress(impl):
 
 @pytest.mark.skipif(sys.version_info < (3, 5), reason="Network decoding requires Py3.5+")
 @pytest.mark.parametrize('payload, expected', [
-    ('d90105a144c0a800641818', ip_network('192.168.0.100/24', False)),
+    ('d90105a144c0a800641818', ip_network(u'192.168.0.100/24', False)),
     ('d90105a15020010db885a3000000008a2e000000001860',
-     ip_network('2001:db8:85a3:0:0:8a2e::/96', False)),
+     ip_network(u'2001:db8:85a3:0:0:8a2e::/96', False)),
 ], ids=[
     'ipv4',
     'ipv6',
--- cbor2-5.2.0.post1/tests/test_encoder.py.orig	2021-12-22 23:33:04.000000000 +0100
+++ cbor2-5.2.0.post1/tests/test_encoder.py	2024-05-19 12:11:16.003106292 +0200
@@ -302,8 +302,8 @@ def test_uuid(impl):
 
 @pytest.mark.skipif(sys.version_info < (3, 3), reason="Address encoding requires Py3.3+")
 @pytest.mark.parametrize('value, expected', [
-    (ip_address('192.10.10.1'), 'd9010444c00a0a01'),
-    (ip_address('2001:db8:85a3::8a2e:370:7334'), 'd901045020010db885a3000000008a2e03707334'),
+    (ip_address(u'192.10.10.1'), 'd9010444c00a0a01'),
+    (ip_address(u'2001:db8:85a3::8a2e:370:7334'), 'd901045020010db885a3000000008a2e03707334'),
 ], ids=[
     'ipv4',
     'ipv6',
@@ -315,8 +315,8 @@ def test_ipaddress(impl, value, expected
 
 @pytest.mark.skipif(sys.version_info < (3, 3), reason="Network encoding requires Py3.3+")
 @pytest.mark.parametrize('value, expected', [
-    (ip_network('192.168.0.100/24', False), 'd90105a144c0a800001818'),
-    (ip_network('2001:db8:85a3:0:0:8a2e::/96', False),
+    (ip_network(u'192.168.0.100/24', False), 'd90105a144c0a800001818'),
+    (ip_network(u'2001:db8:85a3:0:0:8a2e::/96', False),
      'd90105a15020010db885a3000000008a2e000000001860'),
 ], ids=[
     'ipv4',
